core/ui/TagTemplate.tid

```diff
@@ -27,4 +33,4 @@ color:$(foregroundColor)$;
 </span>
 \end
 
-<$macrocall $name="tag-body" colour={{!!color}} palette={{$:/palette}}/>
```

Comment:
No, the newline is still there. Your last two commits are empty. What I did in the past to correct a similar situation was (not sure how detailed you need this to be, ask for more if not clear):

1. Delete the messed up pull request (this one).
2. Edit the file locally, upload it to your fork using git push.
3. Generate a new pull request. Make sure the commit doesn't show the last line as changed. Submit.

HTH.